src/extensions/default/bramble/lib/RemoteCommandHandler.js

```diff
@@ -10,6 +10,7 @@ define(function (require, exports, module) {
     var FileSystem         = brackets.getModule("filesystem/FileSystem");
     var HTMLRewriter       = brackets.getModule("filesystem/impls/filer/lib/HTMLRewriter");
     var SidebarView        = brackets.getModule("project/SidebarView");
+    var PreviewView        = brackets.getModule("iframe-browser");
```

Comment:
Modules that are defined outside an extension (i.e., Brackets "core" modules) can be loaded via `brackets.getModule()`.  However, anything that is defined by this extension (e.g., the `iframe-browser` module) must be loaded with `require()`.

Move this line down to line 29, and change it to use `require()`.